data_engineering_pulumi_components/pipelines/land_to_history.py

```diff
@@ -0,0 +1,29 @@
+from data_engineering_pulumi_components.aws import (
+    Bucket,
+    BucketPutPermissionsArgs,
+)
+from pulumi import ComponentResource
+from pulumi_aws.s3 import BucketLifecycleRuleArgs, BucketVersioningArgs
+
+
+class LandToHistory(ComponentResource):
+    def __init__(self, name: str, role_name_to_grant_put_permission: str) -> None:
+        super().__init__(
+            t="data-engineering-pulumi-components:pipelines:land_to_history",
```

Comment:
This should match the class name:
```suggestion
            t="data-engineering-pulumi-components:pipelines:LandToHistory",
```

Reply:
Agreed